crypto/x509/by_store.c

```diff
@@ -91,21 +108,24 @@ static int cache_objects(X509_LOOKUP *lctx, const char *uri,
             break;
     }
     OSSL_STORE_close(ctx);
+    store->ctx = NULL;
 
     return ok;
 }
 
 
 /* Because OPENSSL_free is a macro and for C type match */
```

Comment:
The comment `/* Because OPENSSL_free is a macro and for C type match */`
should be removed now because it is no more to the point:
The new function does more than deallocation, it also closes the store.